docs/guides/audio-tracks.md

```diff
@@ -1,69 +1,80 @@
+// Add the track to the player's audio track list.
+player.audioTracks().addTrack(track);
+```
 
-### enabled
-If this track should be playing or not. In video.js we only allow one track to be enabled at a time. so if you enable more than one the last one to be enabled will end up being the only one.
+### Listen for a Track Becoming Enabled
+When a new track is enabled (other than the main track) an event is fired on the `AudioTrackList` called `"change"`. You can listen for that event and do something with it.
 
-## Interacting with Audio Tracks
-### Doing something when a track becomes enabled
-When a new track is enabled (other than the main track) an event is fired on the `AudioTrackList` called `change` you can listen to that event and do something with it.
-Here's an example:
 ```js
-// get the current players AudioTrackList object
-let tracks = player.audioTracks();
+// Get the current player's AudioTrackList object.
+var tracks = player.audioTracks();
 
-// listen to the change event
+// Listen to the "change" event.
 tracks.addEventListener('change', function() {
 
-  // print the currently enabled AudioTrack label
-  for (let i = 0; i < tracks.length; i++) {
-    let track = tracks[i];
+  // Log the currently enabled AudioTrack label.
+  for (var i = 0; i < tracks.length; i++) {
+    var track = tracks[i];
 
     if (track.enabled) {
-      console.log(track.label);
+      videojs.log(track.label);
       return;
     }
   }
 });
 ```
 
 ## API
+### `videojs.AudioTrack`
+This class can be used to create new audio track objects. Each property below is available as an option to the `AudioTrack` constructor.
+
+#### `kind`
+One of the audio track types supported by Video.js:
+
+- `"alternative"`: A possible alternative to the main track.
+- `"descriptions"`: An audio description of a video track.
+- `"main"`: The primary audio track for this video.
+- `"main-desc"`: The primary audio track, mixed with audio descriptions.
+- `"translation"`: A translated version of the main audio track.
+- `"commentary"`: Commentary on the primary audio track, e.g. a director's commentary.
+
+Defaults to empty string if `kind` is missing or an invalid `kind` is used.
+
+#### `label`
+The label for the track that will be shown to the user. For example, in a menu that lists the different languages available as alternate audio tracks.
+
+#### `language`
+The valid BCP 47 code for the language of the audio track, e.g. `"en"` for English or `"es"` for Spanish. For supported language translations, please see the [Languages Folder (/lang)](https://github.com/videojs/video.js/tree/master/lang) folder located in the Video.js root.
+
+#### `enabled`
+Whether or not this track should be playing. In Video.js, we only allow one track to be enabled at a time; so, if you enable more than one the last one to be enabled will end up being the only one.
```

Comment:
maybe it would be good to add a note that the spec allows more than one track to be enabled, but safari and most implementations only allow one to be enabled at a time.